pkg/kapp/diffgraph/change_graph_test.go

```diff
@@ -95,6 +95,10 @@ kind: CustomResourceDefinition
 apiVersion: apiextensions.k8s.io/v1
 metadata:
   name: app-config
+spec:
+  group: app-group
+  names:
+    kind: app-kind
```

Comment:
overall looks pretty good. lets add a test in this file that asserts that CRDs and namespaces using are ordered correctly by the default config.

Reply:
Done